subcommand/get-consul-client-ca/command_test.go

```diff
@@ -0,0 +1,388 @@
+package getconsulclientca
+
+import (
+	"crypto"
+	"crypto/x509"
+	"fmt"
+	"github.com/hashicorp/go-discover"
+	"io/ioutil"
+	"log"
+	"net"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/sdk/freeport"
+	"github.com/hashicorp/consul/sdk/testutil"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/hashicorp/consul/tlsutil"
+	"github.com/mitchellh/cli"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRun_FlagsValidation(t *testing.T) {
```

Comment:
- ca-file is not required, as I've mentioned above in case you're using HTTP
- server-addr will default to `127.0.0.1:8500`. Added this to the flag description
- added test for the invalid log level